### PR TITLE
Clarification in README.md Regarding 'bundle install' Execution #1338

### LINKED ISSUE #1338: Clarification in README.md Regarding 'bundle install' Execution
### Issue:
The **README.md** file in the Meshery.io repository suggests running `bundle install` explicitly before executing the `make site` command, which might lead to confusion as the `make site` command already takes care of running `bundle install`.

### Proposed Solution:
To avoid any confusion, we can update the **README.md** to reflect that running `bundle install` explicitly is unnecessary when using the `make site` command. The `make site` target in the **Makefile** already handles the `bundle install` internally before serving the site locally.

### We can solve it using two ways:
1) By removing `bundle install` part from **README.md**
2) By removing `bundle install;` part from **Makefile**

_I would love to work on this issue!_

Thank you for your attention to this matter. Let's ensure that the documentation is clear and helpful to all contributors.

---

### PR DESCRIPTION
**Pull request by YashRajSingh-4799**
**Description**
Updated the Readme file regarding "Bundle install" command Execution

This PR fixes #1338

**Notes for Reviewers**
the bundle install command was used at two places in Readme file and Makefile.

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]
@YashRajSingh-4799 Thanks, please update the make site's execution command in readme too.

**Comment by YashRajSingh-4799** [comment]
Hi @thisiskaransgit, 
the makesite command execution is updated now in readme

**Comment by iArchitSharma** [comment]
@YashRajSingh-4799 

> Hi @thisiskaransgit, the makesite command execution is updated now in readme

@YashRajSingh-4799  It looks like there's a misunderstanding in the way you're trying to combine the commands.
The Syntax you are using is invalid, use `&&` to combine two commands
Nevertheless, it might be more effective to specify both commands separately:
 `$ bundle install`
 `$ bundle exec jekyll serve --drafts --livereload`

**Comment by YashRajSingh-4799** [comment]
> @YashRajSingh-4799
> 
> > Hi @thisiskaransgit, the makesite command execution is updated now in readme
> 
> @YashRajSingh-4799 It looks like there's a misunderstanding in the way you're trying to combine the commands. The Syntax you are using is invalid, use `&&` to combine two commands Nevertheless, it might be more effective to specify both commands separately: `$ bundle install` `$ bundle exec jekyll serve --drafts --livereload`



Updated the Makefile commands!!!

Sorry, it was written in the issue that the command is used twice so i just remove the bundle install command. i havnt seen the Makefile command. 
Thank for the review....

File: README.md [hunk 1]
```diff
@@ -102,18 +102,14 @@ Before contributing, please review the [Contribution Flow](https://github.com/me
...
-  ```
 
 ### 3. Serve the site
 
 - Serve the code locally
   ```bash
   $ make site
   ```
-  _Note: From the Makefile, this command is actually running `$ bundle exec jekyll serve --drafts --livereload`_
+  _Note: From the Makefile, this command is actually running `$ bundle install exec jekyll serve --drafts --livereload`_
```
[the remark this answers is not in the document]
Comment (thisiskaransgit) [reply README.md:112]: @YashRajSingh-4799 @iArchitSharma, this was the change I requested

File: Makefile
[the remark this answers is not in the document]
Comment (thisiskaransgit) [reply Makefile:1]: Changes in makefile is not needed @iArchitSharma

**Comment by iArchitSharma** [comment]
I guess its good to go now
Thank you @YashRajSingh-4799

**Review by thisiskaransgit — APPROVED** [review]
@YashRajSingh-4799 Looks good, thank you for the quick fix :tada:

**Review by iArchitSharma — APPROVED** [review]

**Review by iArchitSharma — APPROVED** [review]

**Review by Chadha93 — APPROVED** [review]

**Pull request merged**